BusBoard.ConsoleApp/Program.cs

```diff
@@ -10,18 +11,36 @@ static void Main(string[] args)
     {
         ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
         var tflApiHelper = new TflApiHelper();
+        var postcodeApiHelper = new PostcodeApiHelper();
+
         while (true)
         {
-            Console.WriteLine("Welcome to the BusBoard please enter the bus stop code you wish to check or exit to exit");
+            Console.WriteLine("Welcome to the BusBoard please enter a postcode you wish to check or 'exit' to exit");
             var input = Console.ReadLine();
             if (input == "exit")
             {
                 Console.WriteLine("Thank you for using BusBoard");
                 break;
             }
-            var buses = tflApiHelper.GetBuses(input);
-            var topFive = buses.Take(5).Select(bus => bus.GetBusData());
-            Console.WriteLine(string.Join("\n", topFive));
+
+            var coordinate = postcodeApiHelper.GetPostcodeCoordinates(input);
+            var stopPoints = tflApiHelper.GetStopPoints(coordinate);
+            if (stopPoints == null)
+            {
+                Console.WriteLine("No nearby stops found for that postcode!");
+            }
+
+            else
+            {
+                var buses = new List<BusEntry>();
+                foreach (var stopPoint in stopPoints.Take(2))
+                {
+                    buses.AddRange(tflApiHelper.GetBuses(stopPoint.naptanId));
```

Comment:
Is it clear which buses are from which stops? It might be better for the user if you print the next five buses at each stop. That should also make your code a bit neater 🙂